eng/scripts/splittestdependencies/tests/Generate-Dependency-Test-References-Tests.ps1

```diff
@@ -80,25 +80,21 @@ Describe "Write-Project-Files-To-Matrix" -Tag "UnitTest" {
         @{ TestFolderName = "$PSScriptRoot/testFolder3"; ProjectFiles = "1.txt", "2.txt", "3.txt"; MatrixJsonPath = "$PSScriptRoot/inputs/platform-matrix2.json"; expectOutput="$PSScriptRoot/expectOutputs/expect-matrix3.json" }
     ) {
         Backup-File -targetPath $MatrixJsonPath -backupFolder $TestFolderName
-        Write-Project-Files-To-Matrix -ProjectFiles $ProjectFiles -MatrixJsonPath $MatrixJsonPath -MatrixOutputFolder $TestFolderName -ProjectFileConfigName "OverrideFiles"
```

Comment:
@sima-zhu Did you intend to update these tests? BTW my PR https://github.com/Azure/azure-sdk-for-net/pull/33061 updated the function which broke some tests I was working to update them but haven't finished that yet and I doubt I can do it before my break. If you pull in the latest changes I'd appreciate if you can fix up the expected results based on my changes

Reply:
The test changes in the PR is to fix test failure brought in your PR, so no worries about the fix. I will take care of them in this PR.